rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/RosettaFormatter.xtend

```diff
@@ -236,6 +237,18 @@ class RosettaFormatter extends AbstractRosettaFormatter2 {
 		ele.regionFor.assignment(nameAssignment_1)
 			.prepend[oneSpace]
 	}
+	
+	def dispatch void format(Choice ele, extension IFormattableDocument document) {
+		ele.regionFor.keyword('choice')
+			.append[oneSpace]
+		ele.regionFor.keyword(':')
+			.prepend[noSpace]
+		ele.indentInner(document)
+		ele.attributes.forEach[
+			prepend[newLine]
+			format
+		]
```

Comment:
To have a consistent formatting for newlines, I think we should have the same kind of logic here as for `Data`.

```
		ele.attributes.head
			.prepend[setNewLines(1, 2, 2)]
			.format
		ele.attributes.tail.forEach[
			prepend[newLine]
			format
		]
```
The first case makes sure the first attribute is either prepended with one newline or with two newlines. By default it is two.
The second case makes sure all subsequent attributes are prepended by one newline.